.github/workflows/drupal_rector.yaml

```diff
@@ -0,0 +1,44 @@
+name: rector_examples
+
+on:
+    pull_request: null
+    push:
+        branches:
+            - master
+
+jobs:
+    rector_examples:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v2
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 7.3
+                    coverage: none # disable xdebug, pcov
+                    extensions: "intl"
+
+            # Run standalone install in non-root package, ref https://github.com/rectorphp/rector/issues/732
+            -   run: |
+                    # Install latest Drupal
```

Comment:
When I read `Install latest Drupal` I interpret this as meaning that we install a working site with a database, etc. I understand from reading this that we are downloading the latest Drupal codebase, but I was thrown off.

We can update this later if we come up with better comments.